api-testing/multicloud-api/README.md

```diff
@@ -0,0 +1,27 @@
+# Multi cloud API Automation
+
+**Introduction**
+
+This framework has developed in [JUNIT](https://junit.org/) using [Gradle](https://gradle.org/) Build tool.
```

Comment:
May be You can mention about the compilation and Build process for this module separately?

Reply:
I have already given link.

Reply:
Link is not enough to explain the project compilation & build process. Link will only tell the generic way.

If there are specific steps for this project to compile & build, please capture here itself in the project.

Reply:
I have added build details. Please review.